src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIClientExtensions.cs

```diff
@@ -35,6 +37,21 @@ public static IChatClient AsIChatClient(this ChatClient chatClient) =>
     public static IChatClient AsIChatClient(this OpenAIResponseClient responseClient) =>
         new OpenAIResponseChatClient(responseClient);
 
+    /// <summary>Gets an <see cref="ISpeechToTextClient"/> for use with this <see cref="OpenAIClient"/>.</summary>
+    /// <param name="openAIClient">The client.</param>
+    /// <param name="modelId">The model.</param>
+    /// <returns>An <see cref="ISpeechToTextClient"/> that can be used to transcribe audio via the <see cref="OpenAIClient"/>.</returns>
+    [Experimental("MEAI001")]
+    public static ISpeechToTextClient AsSpeechToTextClient(this OpenAIClient openAIClient, string modelId) =>
+        new OpenAISpeechToTextClient(openAIClient, modelId);
```

Comment:
Let's delete this one. We're removing the corresponding ones for chat and embeddings as well.